akka-docs/src/main/paradox/futures.md

```diff
@@ -91,15 +91,78 @@ When using non-blocking it is better to use the `mapTo` method to safely try to
 The `mapTo` method will return a new `Future` that contains the result if the cast was successful,
 or a `ClassCastException` if not. Handling `Exception`s will be discussed further within this documentation.
 
-@@@ 
+@@@
+
+## Use the pipe pattern
+
+Another useful message-transfer pattern is "pipe", which is to send the result of `Future` to another actor, upon completion of the `Future`.
```

Comment:
> Another useful message-transfer pattern is

I'm going to re-write the previous section as "Use the ask pattern", that's why I said "Another ... pattern" here. The previous section would be re-written in a separate PR.